sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3224,14 +3230,16 @@ pi_result piEventRelease(pi_event Event) {
       // If the fence associated with this command list has signalled, then
       // Reset the Command List Used in this event and put it back on the
       // available list.
-      if (Event->Queue->RefCount > 0) {
-        Event->Queue->ZeCommandListFenceMapMutex.lock();
+      auto Queue = Event->Queue;
+      if (Queue->RefCount > 0) {
```

Comment:
I wonder if really need this now that we lock the whole Queue

Reply:
Yes, I think we do, See reply to similar comment.